Cargo.toml

```diff
@@ -1,23 +1,37 @@
 [workspace]
-members = ["ceramic-kubo-rpc", "ceramic-one"]
+members = [
+    "core",
+    "event",
+    "ceramic-kubo-rpc",
+    "ceramic-one"
+]
 
 [workspace.dependencies]
 anyhow = "1"
 async-stream = "0.3"
 async-trait = "0.1"
 ceramic-one = { path = "./ceramic-one" }
 dag-jose = "0.1"
```

Comment:
OK, as I understand it we don't need any of the ser/de logic that dag-jose provides. In that case can we remove this dep?

Reply:
Looks like kubo-rpc needs it still